nvi-commons/src/main/java/no/sikt/nva/nvi/common/utils/GraphUtils.java

```diff
@@ -19,6 +19,9 @@
 import org.slf4j.LoggerFactory;
 
 public class GraphUtils {
+    public static final String APPLICATION_JSON = "application/json";
+    public static final String PART_OF_PROPERTY = "https://nva.sikt.no/ontology/publication#partOf";
+    public static final String HAS_PART_PROPERTY = "https://bibsysdev.github.io/src/organization-ontology.ttl#hasPart";
```

Comment:
@brinxmat We need to fix this

Reply:
We should change the datamodel so we can use https://nva.sikt.no/ontology/publication#hasPart